### PR TITLE
Feature/show recipe on react

### PR DESCRIPTION
**Pull request by BogdanLinnik**

### CODE REVIEW

File: app/controllers/api/v1/recipes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class Api::V1::RecipesController < ApplicationController
+
+  def show
+    @recipe = Recipe.find(params[:id])
+    @author = @recipe.user || 'Anonymous'
+    @category = @recipe.category
+    @comments = @recipe.comments
+    unless @recipe.ratings.empty?
```
Comment (illia108) [inline app/controllers/api/v1/recipes_controller.rb]: If you need an average rating for a recipe, we already have a service for this. Please take a look at the `recipe.rb` and `calculate_rating.rb`

File: app/controllers/registered/recipes_controller.rb [hunk 2]
```diff
@@ -40,8 +40,13 @@ def update
...
   def show
     @recipe = Recipe.find(params[:id])
-    @author = @recipe.user&.name || 'Anonymous'
-    save_recently_opened_recipes(@recipe.id)
+    @category = @recipe.category
+    @author = @recipe.user || 'Anonymous'
+    if @recipe.ratings.empty?
+      @ratings = "No ratings for recipe"
+    else
+      @ratings = @recipe.ratings.sum(:rating) / @recipe.ratings.count.to_f
```
Comment (illia108) [inline app/controllers/registered/recipes_controller.rb]: same here

File: app/controllers/api/v1/recipes_controller.rb (hunk 1 above)
Comment (BogdanLinnik) [reply app/controllers/api/v1/recipes_controller.rb]: Fixed

File: app/controllers/registered/recipes_controller.rb (hunk 2 above)
Comment (BogdanLinnik) [reply app/controllers/registered/recipes_controller.rb]: Fixed too

File: app/controllers/admin/recipes_controller.rb [hunk 3]
```diff
@@ -20,9 +20,7 @@ def show
       else
         @rating = current_user.ratings.where(recipe_id:  @recipe.id).first
     end
-    unless @recipe.ratings.empty?
-      @ratings = @recipe.ratings.sum(:rating) / @recipe.ratings.count.to_f
-    end
+    @ratings = @recipe.avarage_rating
```
Comment (illia108) [inline app/controllers/admin/recipes_controller.rb]: `avarage_rating` please rename `average_rating`

**Pull request merged**